app/src/main/java/org/alexcode/snake/MainActivity.java

```diff
@@ -15,7 +16,7 @@
 public class MainActivity extends AppCompatActivity {
     private Button startGame, profile, ranking, settings;
     private String playerName, volume, language, gameMode;
-    private int gamesPlayed, hiScore;
+    int gamesPlayed, hiScore;
```

Comment:
why don't you need this variables private anymore?

Reply:
My mistake. No reason, I will put private again. Thx